### PR TITLE
[PXN-3229] - Remove duplicate total

### PR DESCRIPTION
**Pull request by samantaalbanez**
## Motivación y Contexto
When a payment is made using a debit card the total was getting doubled on the screen. Task: [PXN-3231](https://mercadolibre.atlassian.net/jira/software/c/projects/PXN/boards/5613?modal=detail&selectedIssue=PXN-3231&assignee=61936916c510bc006b1e680f).

## Descripción
Changed the view of the duplicate total:
* The payment is made with a debit card;
* The split that allows using the account balance to complement the payment is enabled;

## Cómo probarlo
* Recharging a cell phone with a debit card using values.

## Screenshots
| Before | After |
|-------|---------|
|Captura de Tela 2022-03-03 às 19 18 08|Captura de Tela 2022-03-03 às 20 23 58
|Captura de Tela 2022-03-03 às 19 18 29|Captura de Tela 2022-03-03 às 20 23 38
|Captura de Tela 2022-03-03 às 19 18 38|Captura de Tela 2022-03-03 às 20 23 46

### CODE REVIEW

**Review by murijr — APPROVED** [review]

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentMethodHeaderViewV2.java [hunk 1]
```diff
@@ -88,4 +89,12 @@ private void showInstallments() {
         titlePager.setVisibility(View.GONE);
         listener.onInstallmentViewUpdated();
     }
+
+    private void configureTitleVisibility(final boolean isDisabled, final boolean hasPayerCost) {
+        if (this.paymentType.equals("debit_card"))
```
Comment (daniivillegas) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentMethodHeaderViewV2.java:94]: En vez de harcodear "debit_card", usaria PaymentTypes.DEBIT_CARD.
Y sacaria el this

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentMethodHeaderViewDefault.java [hunk 2]
```diff
@@ -128,8 +128,10 @@ private void showTitlePager(final boolean isClickable) {
         if (titleView.getVisibility() == VISIBLE) {
             arrow.startAnimation(rotateDown);
         }
-
-        titlePager.setVisibility(VISIBLE);
+        if (this.paymentType.equals("debit_card"))
```
Comment (daniivillegas) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentMethodHeaderViewDefault.java:131]: En vez de harcodear "debit_card", usaria PaymentTypes.DEBIT_CARD.
Y sacaria el this

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentMethodHeaderView.java [hunk 3]
```diff
@@ -65,4 +68,14 @@ public Model(final GoingToModel goingTo, final boolean currentIsExpandable,
             this.nextIsExpandable = nextIsExpandable;
         }
     }
+
+    public void setPaymentTypeAndSplitSelection(String paymentType, final boolean isSelected) {
```
Comment (daniivillegas) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentMethodHeaderView.java:72]: final :P

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentMethodHeaderView.java [hunk 4]
```diff
@@ -65,4 +68,14 @@ public Model(final GoingToModel goingTo, final boolean currentIsExpandable,
...
+        this.paymentType = paymentType;
+        this.splitSelection = isSelected;
+    }
+
+    protected void changeVisibilityTitle(boolean isVisible) {
```
Comment (daniivillegas) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentMethodHeaderView.java:77]: en vez de change le pondria algo como el metodo de arriba setHelperVisibility

**Review by daniivillegas — APPROVED** [review]

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/one_tap/slider/PaymentMethodHeaderAdapter.java [hunk 5]
```diff
@@ -34,6 +34,7 @@ public void updateData(final int currentIndex, final int payerCostSelected,
             data.get(currentIndex);
         paymentMethodDescriptorModelByApplication.update(application);
         final PaymentMethodDescriptorView.Model currentModel = paymentMethodDescriptorModelByApplication.getCurrent();
+        view.setPaymentTypeAndSplitSelection(application.getPaymentMethod().getType(), splitSelectionState.userWantsToSplit());
```
Comment (jvillada31) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/one_tap/slider/PaymentMethodHeaderAdapter.java:37]: 👏

**Review by jvillada31 — APPROVED** [review]

**Review by cardosodiego — APPROVED** [review]

**Review by RoBarriosG — APPROVED** [review]

**Pull request merged**